server/src/modules/rules/getter/sonarr-getter.service.spec.ts

```diff
@@ -442,6 +442,106 @@ describe('SonarrGetterService', () => {
     );
   });
 
+  describe('fileAudioLanguages', () => {
+    let collectionMedia: CollectionMedia;
+    let mockedSonarrApi: SonarrApi;
+    let series: SonarrSeries;
+    let plexLibraryItem: PlexLibraryItem;
+
+    beforeEach(() => {
+      collectionMedia = createCollectionMedia(EPlexDataType.EPISODES);
+      collectionMedia.collection.sonarrSettingsId = 1;
+      plexApi.getMetadata.mockResolvedValue(
+        createPlexMetadata({
+          type: 'show',
+        }),
+      );
+      series = createSonarrSeries();
+      mockedSonarrApi = mockSonarrApi(series);
+      plexLibraryItem = createPlexLibraryItem('episode');
+    });
+
+    it('should return audio languages', async () => {
+      const episodeFile = createSonarrEpisodeFile({
+        mediaInfo: {
+          audioBitrate: 1000,
+          audioChannels: 2,
+          audioCodec: 'AAC',
+          audioLanguages: 'eng',
+          audioStreamCount: 1,
+          videoBitDepth: 8,
+          videoBitrate: 5000,
+          videoCodec: 'H.264',
+          videoFps: 24,
+          resolution: '1920x1080',
+          runTime: '00:42:00',
+          scanType: 'Progressive',
+          subtitles: 'eng',
```

Comment:
I think it would be cleaner to be able to merge in some mediaInfo defined here, as we're only interested in setting the **audioLanguages** property. The remaining properties can be left to faker.